### PR TITLE
I implemented a way to get the latest predictions

### PR DESCRIPTION
**Pull request by HossamBasion**
I also fixed the timing issue, so that it can print in milliseconds

### CODE REVIEW

File: fashionStart.m [hunk 1]
```diff
@@ -281,6 +288,31 @@
...
+%Average the total time by dividing by number of times loop runs
+averageRunTime = averageRunTime / t;
+
+
+
+disp("Average Run Time: ");
+fprintf('%d milliseconds\n',averageRunTime);
+    
+
+disp("Prediction: ");
```
Comment (laurenhiya21) [inline fashionStart.m:312]: I like the idea of seeing the Predictions, but we might not want to print it out every time right now so we can more easily compare the correctness of the networks.

Comment (laurenhiya21) [reply fashionStart.m:312]: We can change this in a future commit however.

**Review by laurenhiya21 — APPROVED** [review]
Looks good!

File: fashionStart.m (hunk 1 above)
Comment (HossamBasion) [reply fashionStart.m:312]: Agreed, I just had it there as an example of how to be used

**Pull request merged**